### ISSUE TITLE
Request .desktop file improvement

### ISSUE THREAD

**Issue by phipolis** [Tympanix/Electorrent#76]
Version 2.1.9 on Fedora 26 / Gnome 3.24.2

From what I understand, the requirements for creating an AppImage include a .desktop file. It would be wonderful if that file could include a MimeType entry so that Electorrent could be set as a handler for magnet links. I looked around the repo and wasn't able to find resources related to generating AppImage builds, including the .desktop file. 

The below is from Transmission Remote GTK's desktop file:
```
MimeType=application/x-bittorrent;x-scheme-handler/magnet;
Categories=Network;FileTransfer;P2P;GTK;
Keywords=p2p;bittorrent;transmission;rpc;
```

**Comment by Tympanix** [comment]
I was not aware of this but apparently the [protocol scheme API](https://electron.atom.io/docs/api/app/#appsetasdefaultprotocolclientprotocol-path-args-macos-windows) only works for macOS and windows. The *AppImage* is being handled by [electron-builder](https://github.com/electron-userland/electron-builder) which is why you could not find source code for this. I will have a look at it and try to improve on this matter.

**Comment by Tympanix** [comment]
I have released [v2.1.10](b68b6b3b9044a161a10d8c234c5fe8f43fd2a234) which adds mime-types, categories and keywords to the *.desktop* file. Please feel free to get back to me if this improves the experience to your satisfaction or further elaboration is needed.
